packages/@secretlint/config-loader/src/index.ts

```diff
@@ -8,9 +8,18 @@ import {
 } from "@secretlint/types";
 import { secretLintProfiler } from "@secretlint/profiler";
 import { SecretLintModuleResolver } from "./SecretLintModuleResolver";
-import { moduleInterop } from "@textlint/module-interop";
 import { validateConfig, validateRawConfig } from "@secretlint/config-validator";
 
+export function moduleInterop<T>(moduleExports: T): T {
+    if (moduleExports && (moduleExports as any).__esModule) {
+        return (moduleExports as any).default?.default!;
+    }
+    return moduleExports && (moduleExports as any).default ? (moduleExports as any).default! : moduleExports;
+}
+
+// FIXME: https://github.com/microsoft/TypeScript/issues/43329
+// module: node12 will be replace it
+const _importDynamic = new Function("modulePath", "return import(modulePath)");
```

Comment:
Dynamic Import hack in CommonJS + TypeScript.

- [ ] https://github.com/microsoft/TypeScript/issues/43329 will fix it

Reply:
This workaround should be removed in the future

- CSP Issue
- Performance Issue
- Type Issue